src/ui-config/networksConfig.ts

```diff
@@ -69,6 +69,25 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
     isTestnet: true,
     networkLogoPath: '/icons/networks/ethereum.svg',
   },
+  [ChainId.sepolia]: {
+    name: 'Ethereum Sepolia',
+    publicJsonRPCUrl: [
+      'https://rpc.sepolia.org',
+      'https://rpc2.sepolia.org',
+      'https://rpc.sepolia.online',
+      'https://www.sepoliarpc.space',
+    ],
+    // publicJsonRPCWSUrl: 'wss://eth-goerli.public.blastapi.io',
+    // protocolDataUrl: '',
+    baseUniswapAdapter: '0x0',
+    baseAssetSymbol: 'ETH',
+    wrappedBaseAssetSymbol: 'WETH',
+    baseAssetDecimals: 18,
+    explorerLink: 'https://goerli.etherscan.io',
```

Comment:
Should be https://sepolia.etherscan.io/

Reply:
`46a5acb60ae6688ce553a713c84135ba259ccd67`